service-worker.js

```diff
@@ -0,0 +1,33 @@
+---
+layout: null
+---
+const version = "{{ site.time | date:'%Y%m%d%H%M%S'}}";
+const cacheName = `cache-${version}`;
+
+// A list of local resources we always want to be cached.
+const cacheURLs = [
```

Comment:
I think this current list is just going to cache the HTML, and not the actual resources (JS/CSS). Is this intentional? I'd expect the site would just look broken offline unless we have inline css that covers most things?

Reply:
Good point. I will add the CSS and JS necessary to render the page from cache.